.github/workflows/build.yml

```
@@ -213,8 +223,20 @@ jobs:
         run: |
           kubectl delete job django-migrate-job --ignore-not-found
           kubectl apply -f k8s/jobs/django-migrate-job.yml
-          kubectl wait --for=condition=complete job/django-migrate-job --timeout=120s
+
+          echo "Describing job status..."
+          kubectl describe job django-migrate-job
+          echo "Showing last 50 lines of job logs..."
+          kubectl logs job/django-migrate-job --tail=50
+          echo "Waiting for job to complete..."
+          kubectl wait --for=condition=complete job/django-migrate-job --timeout=300s
           
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)